internal/regexes/regexes.go

```diff
@@ -47,6 +51,16 @@ const (
 )
 
 var (
+	schemes = `(((http|https))://)`
```

Comment:
why are there so many capture groups in this regex?

Reply:
this should be enough:
```suggestion
	schemes = `(http|https)://`
```

Reply:
this was literally just a drag and drop from another area of the code. you're right :sweat_smile: